.github/workflows/run-tests.yml

```diff
@@ -42,13 +47,22 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Cache chianti database
+      uses: actions/cache@v2
+      if: matrix.toxenv != 'build_docs' && matrix.toxenv != 'codestyle'
+      env:
+        # Bump the version number when changing the chianti version to test with
+        cache-name: cache-chianti-dbase-807
```

Comment:
This an environment variable set to the name of the cache, it's only used in line 58 below